stratum/hal/lib/tdi/tdi_sde_helpers.cc

```diff
@@ -403,15 +403,23 @@ ::util::Status GetAllEntries(
   const ::tdi::Device *device = nullptr;
   ::tdi::DevMgr::getInstance().deviceGet(0, &device);
   const auto flags = ::tdi::Flags(0);
-  uint32 entries = 0;
+  tdi_status_t tdi_status;
```

Comment:
In C++, variables should generally be defined near the point at which they are used rather than at the top of the function.

I would suggest deleting this definition and instead making the suggested change at line 427.